### ISSUE TITLE
Can't perfectly center text

### ISSUE THREAD

**Issue by danielnaxon-inmanage** [claviska/SimpleImage#212]
I'm trying to create a hangouts-like profile picture. A simple circle with 2 letters inside (user's initials).

The problem is that the text isn't being centered perfectly. The letters are placed in a different position for every different combination.

This is the code:
```
$image = new SimpleImage();
$image->fromNew(500, 500)
    ->ellipse(250, 250, 500, 500, $color, 'filled')
    ->text('YA', array(
        'fontFile' => '...',
        'size' => 250,
        'color' => 'white',
        'xOffset' => 0,
        'yOffset' => 0,
    ))
    ->toScreen();
```

examples: https://imgur.com/a/Ah3ZG

![uAOCPk5](https://user-images.githubusercontent.com/55639/91433863-eb7b7680-e831-11ea-9ba9-7cc95dc7528a.png)
![r4O93BN](https://user-images.githubusercontent.com/55639/91433864-ec140d00-e831-11ea-8085-418de9eb8620.png)

**Comment by claviska** [comment]
That's interesting. Do you get the same result from other fonts as well?

For anyone looking to troubleshoot, [this is a good place to start](https://github.com/claviska/SimpleImage/blob/master/src/claviska/SimpleImage.php#L939-L941). The general math should be correct, so it's possible GD is drawing the letters like this. Not sure if there's a way to fix that off the top of my head.

**Comment by danielnaxon-inmanage** [comment]
tried it also with open sans - same result

בתאריך 2 בנוב׳ 2017 16:36,‏ "Cory LaViska" <notifications@github.com> כתב:

**Comment by brittonk** [comment]
This is an old post but for anyone encountering this, I'd assume the problem is that you're not using a monospace font (letters all having the same width)

Have you tried that? Open Sans isn't monospaced.

Here is a list of 10 monospace fonts.
https://www.typewolf.com/top-10-monospaced-fonts

**Comment by maPer77** [comment]
**Solved** #266
